### PR TITLE
VEGA-3192: [FE] Аудит зависимостей

### PR DESCRIPTION
**Pull request by maksim-kononov-csssr**
## Задача

Задача в общей Jira: https://jira.vegaspace.tk/browse/VEGA-3192
Задача в Jira CSSSR: https://jira.csssr.io/browse/VEGA-914

### Description
Аудит зависимостей

### Checks
- [x] Jira link attached: [VEGA-3192](https://jira.vegaspace.tk/browse/VEGA-3192)
- [ ] Possible Associated Jira links:
- [x] Personal Vega Instance link attached: http://mk-example-csssr-vega-builder.code013.org/
- [ ] Jira tasks for technical debt were created (if needed)

### Type
- [ ] Feature
- [ ] Alpha Feature ([Alpha] MR/PR Title)
- [ ] Bug Fix
- [ ] Test
- [x] Refactoring

### PR/MR Requirements
- [x] PR/MR title meet requirements/convention
- [x] Target branch is correct
- [x] PR/MR branch was rebased at correct branch
- [x] Diff was checked
- [x] Configuration files were checked at test repositories
- [x] [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)

### General
- [x] Exploratory Tested at Personal Vega Instance
- [x] Existing E2E Tests done, Screenshot attached
- [x] Existing Integration Tests done, Screenshot attached, Jenkins link:
- [x] Existing Unit Tests done, Screenshot attached
- [ ] New E2E Tests developed
- [ ] New Integration Tests developed
- [ ] New Unit Tests developed
- [ ] API documentation was fully and correctly updated

### E2E Suite Jenkins link
http://jenkins-remote-ci-gpn-vega-builder.code013.org:8081/job/STAND%20TOOLBOX/job/scenario-tests-on-demand/347/

### E2E Suite Screenshot
E2E_test

### Integration Tests Screenshot
_{Screenshot}_

### Unit Tests Screenshot
Unit_test_vega-sp

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,10 +51,8 @@
     "react-final-form": "^6.5.1",
     "react-final-form-arrays": "^3.1.2",
     "react-router-dom": "^5.2.0",
-    "uuid": "^8.3.2"
-  },
-  "devDependencies": {
-    "@gpn-prototypes/frontend-configs": "^1.2.12",
+    "uuid": "^8.3.2",
+    "@gpn-prototypes/frontend-configs": "1.2.16-alpha.2",
```
Comment (hitmanet) [inline package.json:55]: а что в альфа-версии?

File: src/ui/features/projects/ProjectForm/steps/DocumentStep/SidebarForm/SidebarForm.tsx [hunk 2]
```diff
@@ -16,8 +16,8 @@ import { SidebarItem } from './SidebarItem';
 import { getUploadStatistics } from './utils';
 
 type SidebarFormProps = {
-  isOpen: boolean;
-  isMinimized: boolean;
+  isOpen: boolean | undefined;
```
Comment (hitmanet) [inline src/ui/features/projects/ProjectForm/steps/DocumentStep/SidebarForm/SidebarForm.tsx:19]: Можно ж просто optional поле сделать без явного указания undefined

**Review by hitmanet — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (maksim-kononov-csssr) [reply package.json:55]: Изменил @gpn-prototypes/frontend-configs и пока для теста использую альфа версию

File: src/ui/features/projects/ProjectForm/steps/DocumentStep/SidebarForm/SidebarForm.tsx (hunk 2 above)
Comment (maksim-kononov-csssr) [reply src/ui/features/projects/ProjectForm/steps/DocumentStep/SidebarForm/SidebarForm.tsx]: Исправил

File: package.json (hunk 1 above)
Comment (alexeyandriyash) [reply package.json:55]: does alfa version still actual

File: src/ui/features/projects/ProjectForm/steps/DocumentStep/SidebarForm/SidebarForm.tsx [hunk 3]
```diff
@@ -16,8 +16,8 @@ import { SidebarItem } from './SidebarItem';
 import { getUploadStatistics } from './utils';
 
 type SidebarFormProps = {
-  isOpen: boolean;
-  isMinimized: boolean;
+  isOpen?: boolean | undefined;
```
Comment (alekskond) [inline src/ui/features/projects/ProjectForm/steps/DocumentStep/SidebarForm/SidebarForm.tsx:19]: В отдельный PR

Comment (fixmylie) [reply src/ui/features/projects/ProjectForm/steps/DocumentStep/SidebarForm/SidebarForm.tsx:19]: а почему не `isOpen?: boolean`

**Review by fixmylie — APPROVED** [review]

**Review by sergavshin — APPROVED** [review]

**Pull request merged**